config.go

```diff
@@ -1094,8 +1105,17 @@ func loadConfig(appName string) (*config, []string, error) {
 	cfg.RPCListeners = normalizeAddresses(cfg.RPCListeners,
 		cfg.params.rpcPort)
 
+	// The authtype config must be one of "basic" or "clientcert".
```

Comment:
Probably better off creating constants for the expected options for `authtype`. Noticed the strings are repeated for multiple checks.

Reply:
Looks like this was done via `authTypeBasic` and `authTypeClientCert`.